contracts/core/lib/auction-price-libraries/LinearAuctionPriceCurve.sol

```diff
@@ -28,27 +28,66 @@ import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
 contract LinearAuctionPriceCurve {
     using SafeMath for uint256;
 
+    uint256 public priceDenominator;
+
+    /*
+     * Declare price denominator (or precision) of price curve
+     *
+     * @param  _priceDenominator        The priceDenominator you want this library to always return
+     */
+    constructor(
+        uint256 _priceDenominator
+    )
+        public
+    {
+        // Set price to be returned by library
+        priceDenominator = _priceDenominator;
+    }
+
+    /*
+     * Validate any auction parameters that have library-specific restrictions
+     *
+     * @param  -- Unused auction time to pivot param to conform to IAuctionPriceCurve --
```

Comment:
This `unused` construct is a bit wonky. Can we use a struct instead?

Reply:
+1